protocol/avdl/keybase1/contacts.avdl

```diff
@@ -0,0 +1,44 @@
+@namespace("keybase.1")
+protocol contacts {
+    import idl "common.avdl";
+
+    // Contact syncing / resolving
+    // ---------------------------
+
+    record ContactComponent {
+        // One of:
+        union { null, RawPhoneNumber } phoneNumber;
+        union { null, EmailAddress } email;
+        string label; // "", "Home", "Work"
+    }
+    record Contact {
+        string name; // "Bob Goodman"
```

Comment:
I think this should split to `first` and `last` fields if the native import has it separated that way so we can sort the output to match the user's normal view of their contact list